src/test/ui/panic-brace.rs

```diff
@@ -0,0 +1,18 @@
+// build-pass (FIXME(62277): should be check-pass)
+
+const C: &str = "abc {}";
+static S: &str = "{bla}";
+
+#[allow(unreachable_code)]
+fn main() {
```

Comment:
Can you add a test that ensures this lint does not fire on any arbitrary macro that's named `panic!` (e.g. a custom-defined one)?

Reply:
Good idea. Done. :)